test/e2e/storage/testsuites/snapshottable.go

```diff
@@ -252,9 +252,6 @@ func (s *snapshottableTestSuite) DefineTests(driver storageframework.TestDriver,
 				}
 
 				restoredPod = StartInPodWithVolumeSource(ctx, cs, volSrc, restoredPVC.Namespace, "restored-pvc-tester", "sleep 300", config.ClientNodeSelection)
-				cleanupSteps = append(cleanupSteps, func() {
-					StopPod(ctx, cs, restoredPod)
-				})
```

Comment:
I think some `defer` / `cleanupSteps` to delete the pod is still necessary, so the pod is deleted in case any of `framework.ExpectNoError` below fails.  But it should not wait for the pod to be deleted, a simple `cs.CoreV1().Pods(restoredPod.Namespace).Delete(...)` + ignore `NotFound` error could be enough.

Reply:
added e2epod.DeletePodWithWait, is it ok?

Reply:
Yes, it is